hawc/apps/assessment/models.py

```diff
@@ -308,83 +309,43 @@ class Meta:
     def __str__(self):
         return f"{self.name} ({self.year})"
 
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        AssessmentPermissions.clear_cache(self.id)
```

Comment:
Most of our cache updates are done as signals. This would also let it run on `delete`

Reply:
Oh sorry, I dont have a super-strong opinion on this. I think if the change introduces side-effects or is complex, it may make sense to change to a signal.  I've read some recent opinionated blog posts that suggested sometimes just overriding the save is cleaner than signals but I could see it argued either way.

I'm not worried about cleaning up the cache w/ deletes.

I can move to a signal if you'd prefer, just let me know...

Reply:
It seems like just busting a cache based on a single model is kind of model-specific w/ few side effects, which is probably why I put it here...

Reply:
I can see it going either way, I was mostly just curious of your thoughts on it. I'm fine with it in its current state!